app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -23,11 +23,16 @@
             - else
                 = f.govuk_url_field :application_link, label: { size: "s" }, form_group: { classes: "optional-field" }
           - else
-            = govuk_notification_banner title: t("banners.important") do |banner|
-              - banner.add_heading text: t(".notification.heading")
-              = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
+            - if current_variant?(:"2021_04_application_preview_test", :banner_link)
+              = govuk_notification_banner title: t("banners.important") do |banner|
+                - banner.add_heading text: t(".notification.heading")
+                = t(".notification.description.content_html", link: govuk_link_to(t(".notification.description.link"), page_path("job-application-preview"), target: "_blank"))
+            - if current_variant?(:"2021_04_application_preview_test", :banner_no_link)
+              = govuk_notification_banner title: t("banners.important") do |banner|
+                - banner.add_heading text: t(".notification.heading")
+                p = t(".notification.description.content_html", link: "")
```

Comment:
This still doesn't make sense to me. Why are we passing in a blank link?

Reply:
I've made some changes that will hopefully make this a bit clearer. I've removed the blank link.

Lemme know what you think.

To clarify, there's now three variants.

`banner_link`: When this variant is applied, we want to display the banner with a link to the preview page. When this is applied we also don't want a link in the radio label. 

`banner_no_link`: When this variant is applied, we want to display the banner without a link to the preview page. When this is applied we also don't want a link in the radio label. 

`label_link`: When this variant is applied, we don't want to display the banner. When this variant is applied, we **do** want a link to be provided in the radio label.

Reply:
I don't think you've pushed those changes, because it still looks the same to me